data/brands/amenity/hospital.json

```diff
@@ -36,10 +36,12 @@
     {
       "displayName": "Cruz Roja",
       "id": "cruzroja-abc639",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["ch"]},
```

Comment:
Hi. "ch" stands for Switzerland. You should replace "ch" by "es". "es" stands for Spain. 

Details : https://en.wikipedia.org/wiki/List_of_countries_by_United_Nations_geoscheme